### DISCUSSION TITLE
posts/033-summarize-2024/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#20] [Announcements]
# posts/033-summarize-2024/

只有 //追尋何物 纔能讓我覺得我還活着

https://example.com

**Comment by erikopak-cmyk** [comment]
學習總是痛苦的，且不知道痛苦何時會結束。
但倘若捱過去，會成爲一個非常強大的人，，，

**Reply by PermeationLoop** [reply]
物質的快樂是短暫的，而精神的快樂是永久的，，，

我想要在學習的痛苦中獲得快樂，我這輩子是不是🈶️了，，，
